### ISSUE TITLE
MP: No Briefing on Mission load

### ISSUE THREAD

**Issue by Tscheche** [acemod/ACE3#1982]
ACE3 Version: 3.2.0

**Mods:**
* @cba_a3
* @ace3
* @Task Force Radio
* @ BW Mod
* @ AllinArmamappack
* @ RHS USA/RF

**Placed ACE3 Modules:**
None

**Description:**
On Mission Load there does not appear the Map with the briefing but loading instantly into the mission. This gets an issue where you need to act fast on respawn.

I nailed the issue down to ACE after a clean installation with only ace active on the Server

**Steps to reproduce:**
After installing ACE start any MP Mission

**Where did the issue occur?**
Dedicated Server

**RPT log file:**
Unfortunaly the Serverprovider dont grant me the acces to the userprofile

**Comment by alganthe** [comment]
I can't reproduce the issue, can you provide a repro mission to make sure it's ace and not the mission itself or a server setting ?

**Comment by SilentSpike** [comment]
I feel like there's no possible way ace is causing this, but I've seen weirder bugs before. I'd definitely check it isn't a server setting or something.

**Comment by Tscheche** [comment]
Honestly Im not sure either. The issue is I testet it with a clean install on escape from stratis. And briefing worked. Installed cba and ace briefing gone. I got this on other missions too so I ruled the the missions out. Left wirh ace and cba. Unfortunanaly I dont have any direct acess to the server besides on the arma root files via ftp or via webinterface of nitrado (provider of the server). I will reload ace to rule out a data corruption and test it with a clean client installation tomorrow.
